ClientApp/src/AppRoutes.js

```diff
@@ -5,11 +5,12 @@ import { Fundraisers } from "./components/fundraiser/Fundraisers";
 import { Donate } from "./components/Donate";
 import { Dashboard } from "./components/user/Dashboard";
 import { CreateFundraisers } from "./components/fundraiser/CreateFundraiser";
+import { Settings } from "./components/Settings";
```

Comment:
This module can't be found in ./components/Settings. You have Settings.jsx in ../src